startUp.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+import time
+import settings
+
+def main(): 
+    fileName = open(sys.argv[1])
+    programs = fileName.read().splitlines()
+    fileName.close()
+    
+    numberOfPrograms = len(programs)
```

Comment:
I'd say this variable probably isn't necessary, just using len(programs) is probably more clear